addons/stock/models/stock_picking.py

```diff
@@ -835,6 +841,13 @@ def _pre_action_done_hook(self):
             pickings_to_backorder = self._check_backorder()
             if pickings_to_backorder:
                 return pickings_to_backorder._action_generate_backorder_wizard()
+        
+        # Zero Quantity Count Wizard
+        if self.env.user.has_group('stock.group_stock_zero_quantity_count') and not self.env.context.get('skip_zqc'):
+            empty_locations = self._get_empty_locations()
```

Comment:
self._check_zqc for consistency with _check_backorder and check_immediate
also the others _check methods return a picking recordset, maybe you should do that here too